src/WebJobs.Script/Diagnostics/ApplicationInsightsSdkVersionProvider.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Azure.WebJobs.Logging.ApplicationInsights;
+
+namespace Microsoft.Azure.WebJobs.Script.Diagnostics
+{
+    internal class ApplicationInsightsSdkVersionProvider : ISdkVersionProvider
```

Comment:
This is a new class we introduced for setting the App Insights SdkVersion. This is used by their metrics to determine client usage. With auto-tracking there were a couple of new places this needed to be set, so this simplifies things when we need to override it.